app/views/notifications/index.html.erb

```diff
@@ -0,0 +1,28 @@
+<table id="notifications" class="generic_datatable table table-striped table-bordered">
+  <caption><h3><%= @page_title %></h3></caption>
+  <thead>
+    <tr>
+      <th>Status</th>
+      <th>Event</th>
+      <th>Start at</th>
+      <th>Channels</th>
+      <th colspan="1"></th>
+    </tr>
+  </thead>
+I need to allow info only Notifications
+  <tbody>
+    <% @notifications.each do |notification| %>
+      <tr>
+        <td><%= notification.status %></td>
+        <td><%= notification.event ? (link_to notification.event.title, notification.event) : 'None' %></td>
+        <td><%= notification.start_at %></td>
+        <td><%= notification.channels %></td>
+        <td><%= link_to 'Edit', edit_notification_path(notification) %></td>
+      </tr>
+    <% end %>
+  </tbody>
+</table>
+
+<br>
+
+<%= link_to 'New Notification', new_notification_path %>
```

Comment:
Actually they all already do, as it's in the application layout.
https://github.com/ReadyResponder/ReadyResponder/blob/development/app/views/layouts/application.html.erb#L18-L19
Sidebars for everybody !